httpcore5/src/test/java/org/apache/hc/core5/reactor/TestSingleCoreIOReactor.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * ====================================================================
```

Comment:
@kkewwei I am not sure what this test case actually tests. It does not seem to test much of `SingleCoreIOReactor`, just a single reflective call. Drop it.